examples/btc-to-evm-fungible-transfer/src/transfer.p2tr.ts

```diff
@@ -0,0 +1,105 @@
+import type { BaseTransferParams } from "@buildwithsygma/btc";
+import {
+  createBitcoinFungibleTransfer,
+  TypeOfAddress,
+} from "@buildwithsygma/btc";
+import { BIP32Factory } from "bip32";
+import { mnemonicToSeed } from "bip39";
+import { crypto, initEccLib, networks } from "bitcoinjs-lib";
+import { toXOnly } from "bitcoinjs-lib/src/psbt/bip371";
+import dotenv from "dotenv";
+import * as tinysecp from "tiny-secp256k1";
+
+import { broadcastTransaction, getFeeEstimates } from "./utils.js";
+
+dotenv.config();
+
+const SOURCE_DOMAIN_CAIPID = process.env.SOURCE_DOMAIN_CAIPID;
+const DESTINATION_ADDRESS = process.env.DESTINATION_ADDRESS;
+const DESTINATION_DOMAIN_CHAIN_ID = Number(
+  process.env.DESTINATION_DOMAIN_CHAIN_ID,
+);
+const RESOURCE_ID = process.env.RESOURCE_ID;
+const BLOCKSTREAM_URL = process.env.BLOCKSTREAM_URL;
+const EXPLORER_URL = process.env.EXPLORER_URL;
+const MNEMONIC = process.env.MNEMONIC;
+const UTXO_TX_ID = process.env.UTXO_TX_ID;
+const UTXO_AMOUNT = Number(process.env.UTXO_AMOUNT);
+const UTXO_OUTPUT_INDEX = Number(process.env.UTXO_OUTPUT_INDEX);
+const DERIVATION_PATH = process.env.DERIVATION_PATH;
+const CHANGE_ADDRESS = process.env.CHANGE_ADDRESS;
+
+if (
+  !SOURCE_DOMAIN_CAIPID ||
+  !DESTINATION_ADDRESS ||
+  !DESTINATION_DOMAIN_CHAIN_ID ||
+  !RESOURCE_ID ||
+  !MNEMONIC ||
+  !UTXO_TX_ID ||
+  !UTXO_AMOUNT ||
+  !UTXO_OUTPUT_INDEX ||
+  !DERIVATION_PATH ||
+  !CHANGE_ADDRESS ||
+  !BLOCKSTREAM_URL
+) {
+  throw new Error(
+    "Please provided needed env variables needed into the .env file",
+  );
+}
+
+initEccLib(tinysecp);
+const bip32 = BIP32Factory(tinysecp);
```

Comment:
Is there a reason for having this variable defined outside of `btcToEvmTransfer`

Reply:
we don't want to handle wallets inside the SDK. This is a similar approach of having this `const wallet = new Wallet(privateKey ?? "", ethersWeb3Provider);` in the evm examples

Reply:
then we can move this variable inside the function